src/docs/release/breaking-changes/index.md

```diff
@@ -76,3 +77,4 @@ release, and listed in alphabetical order:
 [The Route Transition record and Transition delegate updates]: /docs/release/breaking-changes/route-transition-record-and-transition-delegate
 [ThemeData's accent properties]: /docs/release/breaking-changes/theme-data-accent-properties
 [Material Chip button semantics]: /docs/release/breaking-changes/material-chip-button-semantics
+[Bottom Navigation Item Title]: /docs/release/breaking-changes/bottom-navigation-title-to-label.md
```

Comment:
Delete the ".md" extension. The system converts markdown files to a non-markdown (HTML) format, though we don't use those extensions eithre. This is what is currently breaking the Travis build, btw.

Reply:
Gotcha, thanks for spotting that.